src/System.Runtime/tests/System/GCTests.cs

```diff
@@ -112,12 +119,17 @@ public static void KeepAlive()
 
         private class KeepAliveTest
         {
+            [System.Runtime.CompilerServices.MethodImplAttribute(System.Runtime.CompilerServices.MethodImplOptions.NoInlining)]
+            private static void MakeAndDropDNKA()
+            {
+                new DoNotKeepAliveObject();
```

Comment:
Does it make a difference against original 2 lines?
```c#
var doNotKeepAlive = new DoNotKeepAliveObject();
doNotKeepAlive = null;
```
Just curious, because you kept both lines in the test above ...